grails-app/views/assessment/index.gsp

```diff
@@ -385,37 +389,20 @@
 						</g:form>
 					</div>
 
-					<%--Dialog box for Assessement Plans--%>
-					<div id="assessment-plan" >
-						<g:form controller="assessmentTechnique" method="post" id="${currentImod.id}" params="[learningObjectiveID: currentLearningObjective.id]">
-							<label>
-								<h2>
-									Learning Objective
-								</h2>
-							</label>
-							<br/>
+<%--Dialog box for Assessement Plans--%>
+<div id="assessment-plan" >

```

Comment:
this needs to be tabbed
this `div` should be on the same tab level as the `div` on line `390`